sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2022-2023.bpmn

```diff
@@ -337,7 +337,7 @@ dmnX (comes from dmn)</bpmn:text>
     <bpmn:intermediateThrowEvent id="Event_07l2szk" name="Define Partner Total Income">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=if&#10;  partner1CRAReportedIncome = null&#10;then&#10;  calculatedDataPartner1TotalIncome&#10;else&#10;  partner1CRAReportedIncome" target="calculatedDataPartner1TotalIncome" />
+          <zeebe:output source="=if calculatedDataCurrentYearPartnerIncome = null&#10;then&#10;  if partner1CRAReportedIncome = null&#10;  then&#10;    calculatedDataPartner1TotalIncome&#10;  else&#10;    partner1CRAReportedIncome&#10;else&#10;  calculatedDataCurrentYearPartnerIncome" target="calculatedDataPartner1TotalIncome" />
```

Comment:
The technical AC explains to convert this logic as a flow similar to student income.

![image](https://github.com/user-attachments/assets/7aaa7bd3-5879-43d7-ac61-495dae90a508)

![image](https://github.com/user-attachments/assets/67e30785-516b-4f47-9aeb-2c03fc2e13f4)